src/modules/dashboard/components/dialogs/AdvancedDialog/AdvancedDialog.tsx

```diff
@@ -8,12 +8,12 @@ import IndexModal from '~core/IndexModal';
 import { WizardDialogType, useTransformer } from '~utils/hooks';
 import { useLoggedInUser, Colony } from '~data/index';
 
-import { getAllUserRoles } from '../../../transformers';
+import { getAllUserRoles } from '../../../../transformers';
 import {
   canEnterRecoveryMode,
   hasRoot,
   canArchitect,
-} from '../../../users/checks';
+} from '../../../../users/checks';
```

Comment:
Wonder if we could also in this PR create aliases for paths, to avoid any mistake in "../../.." i know its not part of scope in this issue, but it could reduce risk of bad imports, what you think?

Reply:
It's a good idea. Think it will be cleaner. I will add.

Reply:
I have added 3 new aliases for dialogs, checks, and transformers.

Reply:
I have added an alias for Dialogs, and another 'modules' to help shorten & simplify relative paths for some imports, as suggested by you.